pkg/accounting/accounting.go

```diff
@@ -203,7 +208,7 @@ func (a *Accounting) getPeerBalance(peer swarm.Address) (*PeerBalance, error) {
 	if !ok {
 		// balance not yet in memory, load from state store
 		var balance int64
-		err := a.store.Get(balanceKey(peer), &balance)
+		err := a.store.Get(peerBalanceKey(peer), &balance)
```

Comment:
comment unrelated to this particular diff, but since `reserved` is an `int`, its default value will be `0`. 

and you are anyway ignoring it outside of the `getPeerBalance` func, so you can just skip the setting of the `reserved` value in the structs returned in this function.